config/routes.rb

```diff
@@ -139,6 +139,7 @@ def put(*_args); end
 
     namespace :v2x0, :path => "v2.0" do
       resources :authentications, :only => [:show]
+      resources :sources,         :only => [:index]
       resources :tenants,         :only => [:index, :show]
```

Comment:
apparently my comment didn't come with the review...great. 

Just wanted to validate that it looks like with https://github.com/RedHatInsights/sources-monitor/pull/30 it's not hitting the `/tenants` endpoint anymore - so it's not really needed. I'm ok with leaving it here though since we could potentially use it elsewhere, I just want to make sure I'm reading both of these PRs right.

Reply:
Yes, `/tenant` is not needed after the sources-monitor's PR will be applied